membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -49,71 +48,89 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
   lazy val authenticationService: AuthenticationService = IdentityAuthService
   lazy val metrics = Metrics("AttributesController")
 
+  private def lookup(endpointDescription: String, onSuccessMember: Attributes => Result, onSuccessMemberAndOrContributor: Attributes => Result, onNotFound: Result, doExperiment: Boolean) =
+  {
+    def pickAttributes(identityId: String, prodTest: Boolean)(implicit request: BackendRequest[AnyContent]) = {
+      prodTest match {
+        case true =>
+          val percentageInTest = request.touchpoint.scheduledUpdateVariables.getPercentageTrafficForZuoraLookup
+          request.touchpoint.testAllocator.isInTest(identityId, percentageInTest) match {
+            case true =>
+              attributesFromZuora(identityId, request.touchpoint.zuoraRestService, request.touchpoint.subService)
```

Comment:
we should probably have some logging to tell kibana which path we are following - if there's not already enough